.github/workflows/build_and_test.yml

```diff
@@ -7,6 +7,11 @@ on:
   pull_request:
     branches:
     - master
+  workflow_dispatch:
+    inputs:
+      target:
+        description: 'Target branch to test'
```

Comment:
no big deal but I would call it `Target branch to run`.

Reply:
okay, I'll update.